practicedsa.html

```diff
@@ -133,14 +77,14 @@
                                 class="sr-only">(current)</span></a>
                     </li>
                     <li class="nav-item">
-                        <a class="nav-link font-weight-bold nav-items" href="#about">About</a>
+                        <a class="nav-link font-weight-bold nav-items" href="index.html">About</a>
```

Comment:
It should goto the about section directly so the link should be `index.html#about`.